src/JDA/CoreBundle/Resources/views/Team/team.html.twig

```diff
@@ -71,12 +71,13 @@
 	{% trans %}jda.team.webdeveloper.tyreke{% endtrans %}<br/>
 	{% trans %}jda.team.webdeveloper.kyle{% endtrans %}<br/>
 	{% trans %}jda.team.webdeveloper.jessica{% endtrans %}<br/>
+	Hi Hi Hi Hi Hi
 	</p>
 	
 	<h4>{% trans %}jda.team.contentdevelopment{% endtrans %}</h4>
 	<p>{% trans %}jda.team.content.lead{% endtrans %}<br/>
 	{% trans %}jda.team.content.developer{% endtrans %}<br/>
-	{% trans %}jda.team.content.curation{% endtrans %}</br>
+	{% trans %}jda.team.content.cu ration{% endtrans %}</br>
```

Comment:
Alex,
I deleted the extra space and tested it in my dev server and now looks fine.
I commited the changes and should take effect in the pending pull request
#1003.

Thanks.

Javier.

On Mon, Oct 5, 2015 at 3:38 PM, Alex Horak <notifications@github.com> wrote:

> In src/JDA/CoreBundle/Resources/views/Team/team.html.twig
> <https://github.com/Japan-Digital-Archives/Japan-Digital-Archive/pull/999#discussion_r41206912>
> :
>
> >  	</p>
> >  	
> >  	<h4>{% trans %}jda.team.contentdevelopment{% endtrans %}</h4>
> >  	<p>{% trans %}jda.team.content.lead{% endtrans %}<br/>
> >  	{% trans %}jda.team.content.developer{% endtrans %}<br/>
> > -	{% trans %}jda.team.content.curation{% endtrans %}</br>
> > +	{% trans %}jda.team.content.cu ration{% endtrans %}</br>
>
> Looks like there is an extra space here that should be removed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Japan-Digital-Archives/Japan-Digital-Archive/pull/999/files#r41206912>
> .
>



-- 
Javier Cuan